08/11/21 (20:40:29) | ERROR: [Errno 2] No such file or directory: '/home/maker/.config/bpytop/themes'
Traceback (most recent call last):
  File "/home/maker/.local/lib/python3.9/site-packages/bpytop.py", line 1337, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/maker/.config/bpytop/themes'
09/11/21 (01:46:07) | ERROR: [Errno 2] No such file or directory: '/home/maker/.config/bpytop/themes'
Traceback (most recent call last):
  File "/home/maker/.local/lib/python3.9/site-packages/bpytop.py", line 1337, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/maker/.config/bpytop/themes'
09/11/21 (17:55:14) | ERROR: [Errno 2] No such file or directory: '/home/maker/.config/bpytop/themes'
Traceback (most recent call last):
  File "/home/maker/.local/lib/python3.9/site-packages/bpytop.py", line 1337, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/maker/.config/bpytop/themes'
13/11/21 (03:16:35) | ERROR: [Errno 2] No such file or directory: '/home/maker/.config/bpytop/themes'
Traceback (most recent call last):
  File "/home/maker/.local/lib/python3.9/site-packages/bpytop.py", line 1337, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/maker/.config/bpytop/themes'
13/11/21 (03:16:50) | ERROR: [Errno 2] No such file or directory: '/home/maker/.config/bpytop/themes'
Traceback (most recent call last):
  File "/home/maker/.local/lib/python3.9/site-packages/bpytop.py", line 1337, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/maker/.config/bpytop/themes'
13/11/21 (03:16:58) | ERROR: [Errno 2] No such file or directory: '/home/maker/.config/bpytop/themes'
Traceback (most recent call last):
  File "/home/maker/.local/lib/python3.9/site-packages/bpytop.py", line 1337, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/maker/.config/bpytop/themes'
13/11/21 (03:17:35) | ERROR: [Errno 2] No such file or directory: '/home/maker/.config/bpytop/themes'
Traceback (most recent call last):
  File "/home/maker/.local/lib/python3.9/site-packages/bpytop.py", line 1337, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/maker/.config/bpytop/themes'
14/11/21 (01:03:20) | ERROR: [Errno 2] No such file or directory: '/home/maker/.config/bpytop/themes'
Traceback (most recent call last):
  File "/home/maker/.local/lib/python3.9/site-packages/bpytop.py", line 1337, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/maker/.config/bpytop/themes'
14/11/21 (03:01:35) | ERROR: [Errno 2] No such file or directory: '/home/maker/.config/bpytop/themes'
Traceback (most recent call last):
  File "/home/maker/.local/lib/python3.9/site-packages/bpytop.py", line 1337, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/maker/.config/bpytop/themes'
14/11/21 (03:01:45) | ERROR: [Errno 2] No such file or directory: '/home/maker/.config/bpytop/themes'
Traceback (most recent call last):
  File "/home/maker/.local/lib/python3.9/site-packages/bpytop.py", line 1337, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/maker/.config/bpytop/themes'
14/11/21 (11:31:17) | ERROR: [Errno 2] No such file or directory: '/home/maker/.config/bpytop/themes'
Traceback (most recent call last):
  File "/home/maker/.local/lib/python3.9/site-packages/bpytop.py", line 1337, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/maker/.config/bpytop/themes'
15/11/21 (00:40:50) | ERROR: [Errno 2] No such file or directory: '/home/maker/.config/bpytop/themes'
Traceback (most recent call last):
  File "/home/maker/.local/lib/python3.9/site-packages/bpytop.py", line 1337, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/maker/.config/bpytop/themes'
15/11/21 (10:57:51) | ERROR: [Errno 2] No such file or directory: '/home/maker/.config/bpytop/themes'
Traceback (most recent call last):
  File "/home/maker/.local/lib/python3.9/site-packages/bpytop.py", line 1337, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/maker/.config/bpytop/themes'
15/11/21 (17:02:38) | ERROR: [Errno 2] No such file or directory: '/home/maker/.config/bpytop/themes'
Traceback (most recent call last):
  File "/home/maker/.local/lib/python3.9/site-packages/bpytop.py", line 1337, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/maker/.config/bpytop/themes'
15/11/21 (17:47:50) | ERROR: [Errno 2] No such file or directory: '/home/maker/.config/bpytop/themes'
Traceback (most recent call last):
  File "/home/maker/.local/lib/python3.9/site-packages/bpytop.py", line 1337, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/maker/.config/bpytop/themes'
15/11/21 (21:25:56) | ERROR: [Errno 2] No such file or directory: '/home/maker/.config/bpytop/themes'
Traceback (most recent call last):
  File "/home/maker/.local/lib/python3.9/site-packages/bpytop.py", line 1337, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/maker/.config/bpytop/themes'
16/11/21 (15:49:49) | ERROR: [Errno 2] No such file or directory: '/home/maker/.config/bpytop/themes'
Traceback (most recent call last):
  File "/home/maker/.local/lib/python3.9/site-packages/bpytop.py", line 1337, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/maker/.config/bpytop/themes'
16/11/21 (21:57:34) | ERROR: [Errno 2] No such file or directory: '/home/maker/.config/bpytop/themes'
Traceback (most recent call last):
  File "/home/maker/.local/lib/python3.9/site-packages/bpytop.py", line 1337, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/maker/.config/bpytop/themes'
